src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/CovariantEqualsCheck.java

```diff
@@ -46,8 +46,8 @@
  * that are equal using covariant enum method, and not equal when compared normally.
  * </p>
  * <p>
- * Inspired by <a href="https://cs.nyu.edu/~lharris/papers/findbugsPaper.pdf">
```

Comment:
The author's home page has changed. Old home page is inaccessible. New home page contains updated version of the article. This is why the section number is changed also.